src/raftstore/coprocessor/split_check/half.rs

```diff
@@ -224,14 +222,14 @@ mod tests {
             // Flush for every key so that we can know the exact middle key.
             engine.flush_cf(cf_handle, true).unwrap();
         }
-        runnable.run(SplitCheckTask::new(
+        runnable.run(SplitCheckTask::split_check(
```

Comment:
Why change the name from `new` to `split_check`?

Reply:
Because we add a new SplitCheckTask(`Task::ChangeConfig`), use `new` will be ambiguity.